test/voice_public.test.js

```diff
@@ -131,6 +156,68 @@ describe("Mama Nigeria App", function() {
                         })
                         .run();
                 });
+                it("should not restart (or skip to menu_menu state)", function() {
```

Comment:
Please make this clearer that it's preventing skipping ahead to main menu state by checking the no_restart_states in utils.